.travis.yml

```diff
@@ -1,10 +1,18 @@
 language: node_js
 services: mongodb
 sudo: required
-node_js:
-  - 10
-  - 12
-  - 14
+matrix:
+  fast_finish: true
+  include:
+    - name: Node.js 10
+      node_js: 10
+      env: RUN="yarn test"
+    - name: Node.js 12
+      node_js: 12
+      env: RUN="yarn test"
+    - name: Node.js 14 & lint
+      node_js: 14
+      env: RUN="yarn test && yarn lint"
```

Comment:
really minor thought.

'linting' is often checked before 'test'

That's something developers forget often and it fails really fast.

Reply:
Fair point! Changed in https://github.com/agenda/agenda/pull/1182/commits/15872e77c4b17f42a7c6f4e7b552fc0f317127fc